core/src/sumeragi/main_loop.rs

```diff
@@ -540,8 +595,9 @@ fn handle_role_agnostic_message<F: FaultInjection>(
                         peer_latest_block_view_change_index=?state.latest_block_view_change_index,
                         consensus_latest_block_hash=%block.hash(),
                         consensus_latest_block_view_change_index=%block.header().view_change_index,
-                        "Soft fork occurred: peer in inconsistent state. Currently only reload with wipe of the block storage will help."
+                        "Soft fork occurred: peer is inconsistent state. Rolling back and replacing top block."
```

Comment:
```suggestion
                        "Soft fork: peer in inconsistent state. Rolling back and replacing top block."
```

Should read as "(A) Soft fork (has) (occurred): (and) (the) peer (is) in (an) inconsistent state". The *in* is not optional, as it is the active verb of the clause.